docs/src/main/paradox/running-the-application.md

```diff
@@ -0,0 +1,86 @@
+Running the application
+-----------------------
+
+You can run the Hello World application from the command line or an IDE. The final topic in this guide describes how to run it from IntelliJ IDEA. However, before we get there, let’s take a quick look at the build tool: sbt.
+
+## The build files
+
+sbt uses a build.sbt file to handle the project. This project’s build.sbt file looks like this:
+
+@@snip [build.sbt]($g8root$/build.sbt)
+
+## Running the project
+
+We run the application from a console/terminal window:
+
+1. Enter `./sbt` on OSX/Linux or `sbt.bat` on Windows
+
+sbt downloads project dependencies. The `>` prompt indicates sbt has started in interactive mode.
+
+2. At the sbt prompt, enter `run`.
```

Comment:
We should use sbt-revolver here. That's one of the things that everyone likes when you show it.